src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -1,35 +1,82 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { createMuiTheme, ThemeProvider, makeStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Fab from '@material-ui/core/Fab';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Toolbar from '@material-ui/core/Toolbar';
 
 import Version from '../../../../../package.json';
 import './Banner.css';
 
-const Banner = props => (
-  <div className="heroBanner">
-    <div className="bannerImg"></div>
-    <div className="h1">Telescope</div>
-    <div className="version">v {Version.version}</div>
-    <div className="icon">
-      <svg
-        aria-hidden="true"
-        focusable="false"
-        data-prefix="far"
-        data-icon="arrow-alt-circle-down"
-        role="img"
-        xmlns="http://www.w3.org/2000/svg"
-        viewBox="0 0 512 512"
-      >
-        <path
-          fill="#335a7e"
-          d="M256 8C119 8 8 119 8 256s111 248 248 248 248-111 248-248S393 8 256 8zm0 448c-110.5 0-200-89.5-200-200S145.5 56 256 56s200 89.5 200 200-89.5 200-200 200zm-32-316v116h-67c-10.7 0-16 12.9-8.5 20.5l99 99c4.7 4.7 12.3 4.7 17 0l99-99c7.6-7.6 2.2-20.5-8.5-20.5h-67V140c0-6.6-5.4-12-12-12h-40c-6.6 0-12 5.4-12 12z"
-        ></path>
-      </svg>
+const theme = createMuiTheme();
+
+const useStyles = makeStyles({
```

Comment:
Instead of calling `createMuiTheme()`, here, could we alter this `makeStyles` call to inherit a theme ([like the `myfeeds` component does](https://github.com/Seneca-CDOT/telescope/blob/a38207c1deee36f69deb2972a16facef6dcadfa2/src/frontend/src/pages/myfeeds.js#L7))?

https://github.com/Seneca-CDOT/telescope/blob/a38207c1deee36f69deb2972a16facef6dcadfa2/src/frontend/src/pages/myfeeds.js#L7